### PR TITLE
Create Installation_with_Helm.md

### PR DESCRIPTION
**Pull request by drduke1**
###### Description

Fill in your description here.

###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: deploy/docs/Installation_with_Helm.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+sed 's/\collection'"/my-release/g" > values.yaml
+```
+
+Make any changes to the `values.yaml` file as needed, and run the following to install the chart with the override file.
+
+```bash
+helm install sumologic/sumologic --name my-release --namespace my-namespace -f values.yaml --set sumologic.endpoint=<SUMO_ENDPOINT> --set sumologic.accessId=<SUMO_ACCESS_ID> --set sumologic.accessKey=<SUMO_ACCESS_KEY> 
+```
+
+If you would like to use a different cluster name than the default `kubernetes`, add this to the `helm install` command:
```
Comment (samjsong) [inline deploy/docs/Installation_with_Helm.md]: Can we move this up to line 49, right under the first `helm install` command on line 47? Most users want to specify a cluster name and the feedback we received was that this was too far down and most users have already installed by the time they read this and have to reinstall.

File: deploy/docs/Installation_with_Helm.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+- cluster
+- container
+- deployment
+- host
+- namespace
+- node
+- pod
+- service
+
+The Helm chart installation requires three parameter overrides:
```
Comment (frankreno) [inline deploy/docs/Installation_with_Helm.md]: Can we document the clusterName and externalLabels here or at least before the install steps? It is always missed

File: deploy/docs/Installation_with_Helm.md (hunk 1 above)
Comment (frankreno) [reply deploy/docs/Installation_with_Helm.md]: +1

File: deploy/docs/Installation_with_Helm.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+```bash
+helm install sumologic/sumologic --name my-release --namespace my-namespace -f values.yaml --set sumologic.endpoint=<SUMO_ENDPOINT> --set sumologic.accessId=<SUMO_ACCESS_ID> --set sumologic.accessKey=<SUMO_ACCESS_KEY> 
+```
+
+If you would like to use a different cluster name than the default `kubernetes`, add this to the `helm install` command:
+```
+--set prometheus-operator.prometheus.prometheusSpec.externalLabels.cluster="<my-cluster-name>" --set sumologic.clusterName="<my-cluster-name>"
+```
+
+## Uninstalling the Chart
```
Comment (frankreno) [inline deploy/docs/Installation_with_Helm.md]: Since our process is not yet idempotent we should doc deleting the secret and collector

**Pull request merged**